charts/devices-api/values.yaml

```diff
@@ -0,0 +1,212 @@
+  DEFINITIONS_GRPC_ADDR: device-definitions-api-dev:8086
+  DEVICE_DEFINITION_TOPIC: table.device.definition
+  DEVICE_DEFINITION_METADATA_TOPIC: table.device.definition.metadata
+  BLACKBOOK_API_URL: https://service.blackbookcloud.com/UsedCarWS/UsedCarWS
+  ELASTIC_DEVICE_STATUS_INDEX: device-status-dev-*
+  DIMO_REGISTRY_ADDR: '0x4De1bCf2B7E851E31216fC07989caA902A604784'
+  MONITORING_SERVER_PORT: 8888
+  ENABLE_PRIVILEGES: true
+  TOKEN_EXCHANGE_JWK_KEY_SET_URL: http://dex-roles-rights.dev.svc.cluster.local:5556/keys
+  CONTRACT_EVENT_TOPIC: topic.contract.event
+  VEHICLE_NFT_ADDRESS: '0x90c4d6113ec88dd4bdf12f26db2b3998fd13a144'
+  AUTOPI_NFT_IMAGE: autopi.png
+  MACARON_NFT_IMAGE: macaron.png
+  VINCARIO_API_URL: https://api.vindecoder.eu/3.2
+  CHATGPT_URL: https://api.openai.com/v1/chat/completions
+  AFTERMARKET_DEVICE_CONTRACT_ADDRESS: '0xcf9af64522162da85164a714c23a7705e6e466b3'
+  DIMO_CONTRACT_APIURL: https://contracts-api.dev.dimo.zone/
+  NATS_URL: nats-dev:4222
+  NATS_STREAM_NAME: DD_VALUATION_TASKS
+  NATS_VALUATION_SUBJECT: dd_valuation_tasks
+  NATS_DURABLE_CONSUMER: dd-valuation-task-consumer
+  NATS_ACK_TIMEOUT: 2m
+  SYNTHETIC_DEVICES_ENABLED: true
+  SYNTHETIC_WALLET_GRPC_ADDR: synthetic-wallet-instance-dev:8086
+  DEVICE_FINGERPRINT_TOPIC: topic.device.fingerprint
+  DEVICE_FINGERPRINT_CONSUMER_GROUP: device-fingerprint-vc-issuer
+  DEVICE_DATA_GRPC_ADDR: device-data-api-dev:8086
+  SYNTHETIC_FINGERPRINT_TOPIC: topic.synthetic.fingerprint
+  SYNTHETIC_FINGERPRINT_CONSUMER_GROUP: consumer.synthetic.fingerprint
+service:
+  type: ClusterIP
+  ports:
+    mon-http:
+      port: 8888
+      targetPort: mon-http
+      protocol: TCP
+    http:
+      port: 8080
+      targetPort: http
+      protocol: TCP
+    grpc:
+      port: 8086
+      targetPort: grpc
+      protocol: TCP
+ports:
+  - name: mon-http
+    containerPort: 8888
+    protocol: TCP
+  - name: http
+    containerPort: 8080
+    protocol: TCP
+  - name: grpc
+    containerPort: 8086
+    protocol: TCP
+ingress:
+  enabled: true
+  className: nginx
+  annotations:
+    nginx.ingress.kubernetes.io/auth-tls-secret: ingress/cf-origin-ca
+    nginx.ingress.kubernetes.io/auth-tls-verify-client: 'on'
+    nginx.ingress.kubernetes.io/enable-cors: 'true'
+    nginx.ingress.kubernetes.io/cors-allow-origin: https://app.dev.dimo.zone
+    nginx.ingress.kubernetes.io/limit-rps: '3'
+    external-dns.alpha.kubernetes.io/hostname: vehicle-events-api.dev.dimo.zone
+  hosts:
+    - host: vehicle-events-api.dev.dimo.zone
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+resources:
+  limits:
+    cpu: 250m
+    memory: 384Mi
+  requests:
+    cpu: 100m
+    memory: 192Mi
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+nodeSelector: {}
+tolerations: []
+affinity: {}
+podDisruptionBudget:
+  minAvailable: 0
+kafka:
+  clusterName: kafka-dev-dimo-kafka
+  topics:
```

Comment:
currently I don't think events-api needs any kafka topics to be created in dev or prod, let's remove all these